tests/acceptance-bdd/literature_suggestion.feature

```diff
@@ -0,0 +1,306 @@
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+Feature: Litearature Suggestions
+    A web page to submit articles.
+
+  # arXiv text input
+  Scenario: Writing the correct arXiv id
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I insert <input> in the input box with id arxiv_id
+    Then I should not see the arxiv error message
+
+  Examples: Vertical
+    | input | 1001.4538 | hep-th/9711200 |
+
+  Scenario: Writing the wrong arXiv id
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I insert <input> in the input box with id arxiv_id
+    Then I should see the arxiv error message
+
+  Examples: Vertical
+    | input | hep-th.9711200 |
+
+  # DOI text input
+  Scenario: Writing the correct DOI id
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I insert <input> in the input box with id doi
+    Then I should not see the arxiv error message
+
+  Examples: Vertical
+    | input | 10.1086/305772 |
+
+  Scenario: Writing the wrong DOI id
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I insert <input> in the input box with id doi
+    Then I should see the doi error message
+
+  Examples: Vertical
+    | input | dummy:10.1086/305772 | state-doi
+
+
+  # Thesis date text input
+  Scenario: Writing a correct thesis date
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I click on the button Skip, and fill the form manually
+    And I select the value thesis in the select box with id type_of_doc
+    And I insert <input> in the input box with id thesis_date
+    Then I should not see the date error message
+
+  Examples: Vertical
+    | input | 2016-01 | 2016 |
+
+
+  Scenario: Writing a wrong thesis date
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I click on the button Skip, and fill the form manually
+    And I select the value thesis in the select box with id type_of_doc
+    And I insert <input> in the input box with id thesis_date
+    Then I should see the date error message
+
+  Examples: Vertical
+    | input | wrong | 2016-02-30 | 2016-13 |
+
+
+
+  # URL text input
+  Scenario: Writing a correct url
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I click on the button Skip, and fill the form manually
+    And I insert pdf_url_correct in the input box with id url
```

Comment:
`pytest-bdd` is able to manage 2 kinds of input parameters:

- <First_type>
- {Second_type}

This is the [Gherkin](https://github.com/cucumber/cucumber/tree/master/gherkin) syntax, the variable templates are written using corner braces as _<somevalue>_ when we need to test multiple values or curly braces as _{somevalue}_ when we need to test a single value that has to be parameterized.